### PR TITLE
CORTX-28340: Update cortx-k8s to use new cortx-data/-control images

### PR DESCRIPTION
**Pull request by walterlopatka**
## Description
Update cortx-k8s to use new cortx-data and cortx-control images (to replace the original cortx-all image).

This change includes:
* update the solution.example.yaml file
* update upgrade-cortx-cluster.sh (to use new images)
* update documentation
* update regression test
* update github templates that were previously using the term "cortx-all"

Note: I do not plan to add any upgrade regression testing until the upgrade mechanics settle.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds new functionality)
- [ ] Breaking change (bug fix or new feature that breaks existing functionality)
- [ ] Third-party dependency update
- [ ] Documentation additions or improvements
- [x] Code quality improvements to existing code or test additions/updates

## Applicable issues

- This change fixes an issue: CORTX-28340

## CORTX image version requirements
This change requires CORTX images that support the cortx-data, cortx-server, and cortx-control images, which start with version   790.

This change requires the following images:

- `cortx-data:2.0.0-790`
- `cortx-rgw:2.0.0-790`
- `cortx-control:2.0.0-790`

## How was this tested?
* I ran regression tests, which in include deploy, status, shutdown, start, destroy
* I ran upgrade

## Checklist

- [x] The change is tested and works locally.
- [x] New or changed settings in the solution YAML are documented clearly in the README.md file.
- [x] All commits are signed off and are in agreement with the [CORTX Community DCO and CLA policy](https://github.com/Seagate/cortx/blob/main/doc/dco_cla.md).

If this change addresses a CORTX Jira issue:

- [x] The title of the PR starts with the issue ID (e.g. `CORTX-XXXXX:`)

-----
[View rendered .github/PULL_REQUEST_TEMPLATE.md](https://github.com/walterlopatka/cortx-k8s/blob/CORTX-28340_cortx_control/.github/PULL_REQUEST_TEMPLATE.md)
[View rendered README.md](https://github.com/walterlopatka/cortx-k8s/blob/CORTX-28340_cortx_control/README.md)
[View rendered doc/cortx-aws-k8s-installation.md](https://github.com/walterlopatka/cortx-k8s/blob/CORTX-28340_cortx_control/doc/cortx-aws-k8s-installation.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ If you have direct access to the underlying Kubernetes Nodes in your cluster, CO
 2. Run the upgrade script to patch the CORTX on Kubernetes Deployments using an updated image _(:information_source: You will want to update the `TARGET_IMAGE` variable below to your desired image tag)_. The script will stop all CORTX Pods, update the Deployments, and then re-start the Pods.
 
    ```bash
-   TARGET_IMAGE="ghcr.io/seagate/cortx-all:2.0.0-641-custom-ci"
+   TARGET_IMAGE="ghcr.io/seagate/cortx-data:2.0.0-790"
```
Comment (walterlopatka) [inline README.md:187]: It seems simpler to me to list "cortx-data" as the target image rather than explain that it can be cortx-<anything>.

**Review by osowski — APPROVED** [review]
lgtm

File: k8_cortx_cloud/upgrade-cortx-cloud.sh [hunk 2]
```diff
@@ -157,16 +157,32 @@ cortx_deployments="$(kubectl get deployments --namespace="${NAMESPACE}" --output
...
+        elif [[ "${deployment}" == "cortx-client-"* ]]; then
+            IMAGE="${DATA_IMAGE}"
+        elif [[ "${deployment}" == "cortx-control" ]]; then
+            IMAGE="${CONTROL_IMAGE}"
+        elif [[ "${deployment}" == "cortx-ha" ]]; then
+            IMAGE="${CONTROL_IMAGE}"
         else
-            IMAGE="${UPGRADE_IMAGE}"
+            echo "NO MATCH FOR ${deployment}.  Skipping upgrade of image."
+            continue
```
Comment (keithpine) [inline k8_cortx_cloud/upgrade-cortx-cloud.sh:183]: A `case` statement would simplify this.

Comment (walterlopatka) [reply k8_cortx_cloud/upgrade-cortx-cloud.sh]: I agree... updated!

**Pull request merged**